docs/site/reference.md

```diff
@@ -67,9 +71,9 @@ linkage evaluation phase. The following features are supported:
 
 :   The patient's county.
 
-`TELEPHONE`
+`TELECOM`
 
-:   The patient's telephone number.
+:   The patient's phone, email, fax, or other contact information.
```

Comment:
Awesome that we can support email as a matching feature now!

Reply:
I have a feeling NBS will want separate features for matching on phone vs email, but I'd like to address that in the future.